css/components/card.css

```diff
@@ -0,0 +1,178 @@
+.display-flex-column {
+  display: flex;
+  flex-direction: column;
+}
+.display-flex {
+  display: flex;
+  flex-wrap: wrap;
+}
+.flex-wrap-reverse {
+  flex-wrap: wrap-reverse;
+}
+.flex-items {
+  margin: 1rem 1rem 1rem 0;
+}
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  margin-bottom: 1rem;
+  width: 100%;
+}
+.column-80 {
+  width: calc(80% - 1rem);
+}
+.column-20 {
+  width: 20%;
+}
+.column1 {
+  margin-right: 1rem;
+}
+.box-shadow {
+  box-shadow: rgba(0, 0, 0, 0.4) 0px 2px 4px,
+    rgba(0, 0, 0, 0.3) 0px 7px 13px -3px, rgba(0, 0, 0, 0.2) 0px -3px 0px inset;
+}
+.card-horizontal {
+  width: 100%;
+  border-radius: 4px;
+  color: var(--dark-color);
+  display: flex;
+  border: 1px solid var(--border-color);
+}
+.card-horizontal img {
+  width: 150px;
+  height: 200px;
+}
+
+.card-horizontal .text-container {
+  width: calc(100% - 111px);
+}
+.text-container {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  background-color: var(--white-color);
+}
+
+.card-vertical {
+  display: flex;
+  flex-direction: column;
+  max-width: 270px;
+  width: 100%;
+  position: relative;
+}
+
+.card-vertical img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+.price-info {
+  padding: 8px 0;
+}
+.price-info span {
+  margin-right: 1rem;
+}
+.price {
+  font-weight: 600;
+  font-size: 1.2rem;
+}
+.offer {
+  color: var(--success-color);
+}
+.text-strike-through {
+  text-decoration: line-through;
+  color: #757575;
+}
+
+.card-horizontal .badge {
+  position: absolute;
+  top: 0;
+  left: 0;
+  border-radius: 4px;
+  color: var(--white-color);
+  background-color: var(--primary-color);
+}
+
+.flex-grow {
+  flex-grow: 1;
+}
+
+.overlay-text {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+  background-color: rgb(234, 234, 236, 0.6);
+}
+.overlay-container {
+  opacity: 0.9;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+.text-container i {
+  color: var(--white-color);
+  font-size: 1.5rem;
+  margin-right: 5px;
+}
+
+.brand-info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.brand-info i {
+  color: #cccccc;
+  cursor: pointer;
+  font-size: 2rem;
+  padding: 8px;
+}
+.brand-info i:hover {
+  border-radius: 50%;
+  background-color: #f1f5f9;
+}
+
+#close-icon {
```

Comment:
I suggest to make use of classes rather than id's for lower specificity and reusability

Reply:
Thanks for your suggestion